tests/virt_autotest/reboot_and_wait_up.pm

```diff
@@ -22,6 +22,18 @@ sub reboot_and_wait_up {
     my $self           = shift;
     my $reboot_timeout = shift;
 
+    if (check_var('ARCH', 's390x')) {
+        record_info('INFO', 'Reboot LPAR');
+        #Type reboot
+        type_string("reboot\n");
```

Comment:
If possible use `power_action()` from `power_action_utils` package.

Reply:
Cool, power_action() works to me. thank you so much for your suggestion. Now, do code changed about this and verification run. Update PR with this changed ASAP.